python/iceberg/api/expressions/inclusive_metrics_evaluator.py

```diff
@@ -52,13 +53,13 @@ def __init__(self, expr, schema, struct):
         self.struct = struct
 
     def eval(self, file):
-        if file.record_count <= 0:
+        if file.record_count() <= 0:
```

Comment:
Are these changes related to the case sensitivity or just a bug?

Reply:
both items were/are bugs, that were fixed between the initial commit and when I started the case-sensitivity backport

Reply:
ok, this is trivial, so I'll merge, but best to split those out and make smaller commits going forward (where possible).